### PR TITLE
refactor: Changes for new benchmark setup

### PR DESCRIPTION
**Pull request by r-brink**
Renaming elements in the Polars repository to reflect the changes in our new benchmarking set up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,11 +106,11 @@ shape: (5, 8)
 >>> ## OPTION 1
 >>> # run SQL queries on frame-level
 >>> df.sql("""
-...	SELECT species,
-...	  AVG(sepal_length) AS avg_sepal_length
-...	FROM self
-...	GROUP BY species
-...	""").collect()
+... SELECT species,
```
Comment (ritchie46) [inline README.md:109]: I think this was unintended due to your editor? I don't think we should change this.

Comment (r-brink) [reply README.md:109]: @ritchie46 what about the `pl.sql()` example at line 131? Shall I align that example with this one while I am at it?

**Review by ritchie46 — APPROVED** [review]

**Comment by ritchie46** [comment]
@r-brink can you rebase?

**Comment by ritchie46** [comment]
Can you open a new PR? :see_no_evil:

**Pull request closed without merging**